static/app/views/insights/browser/webVitals/components/webVitalMetersWithIssues.spec.tsx

```diff
@@ -0,0 +1,71 @@
+import {OrganizationFixture} from 'sentry-fixture/organization';
+import {PageFilterStateFixture} from 'sentry-fixture/pageFilters';
+
+import {render, screen} from 'sentry-test/reactTestingLibrary';
+
+import {useLocation} from 'sentry/utils/useLocation';
+import usePageFilters from 'sentry/utils/usePageFilters';
+import WebVitalMetersWithIssues, {
+  type ProjectData,
+} from 'sentry/views/insights/browser/webVitals/components/webVitalMetersWithIssues';
+import type {ProjectScore} from 'sentry/views/insights/browser/webVitals/types';
+
+jest.mock('sentry/utils/useLocation');
+jest.mock('sentry/utils/usePageFilters');
+
+describe('WebVitalMetersWithIssues', function () {
+  const organization = OrganizationFixture();
+  const projectScore: ProjectScore = {
+    lcpScore: 100,
+    fcpScore: 100,
+    clsScore: 100,
+    ttfbScore: 100,
+    inpScore: 100,
+  };
+  const projectData: ProjectData[] = [];
+  let issuesMock: jest.Mock;
+
+  beforeEach(function () {
+    jest.mocked(useLocation).mockReturnValue({
+      pathname: '',
+      search: '',
+      query: {},
+      hash: '',
+      state: undefined,
+      action: 'PUSH',
+      key: '',
+    });
+    jest.mocked(usePageFilters).mockReturnValue(PageFilterStateFixture());
+    issuesMock = MockApiClient.addMockResponse({
+      method: 'GET',
+      url: '/organizations/org-slug/issues/',
+      body: [],
+    });
```

Comment:
I'm not sure what the state of this is, but I think it's possible to use router mocks to set these values, would you mind checking [the dev docs](https://develop.sentry.dev/frontend/using-rtl/#use-our-built-in-contexts)?